tools/testing/gatekeeper/required-tests.yaml

```diff
@@ -15,10 +15,9 @@ paths:
   # CI
   - "^ci/openshift-ci/": []
   - "^\\.github/workflows/": []
+  - "\\.md$": ["static"]
   # TODO: Expand filters
-  # Documentation
   #- "\\.rst$": ["build"]
```

Comment:
Could you please remove the `rst` as well? They are not used in this project so there is no need to keep it here...